tests/CMakeLists.txt

```diff
@@ -426,6 +426,6 @@ set_property(TEST ${TEST_TARGET} PROPERTY ENVIRONMENT "LLVM_PROFILE_FILE=${TEST_
 
 set(TEST_TARGET test-backend-ops)
 add_executable(${TEST_TARGET} ${TEST_TARGET}.cpp)
-target_link_libraries(${TEST_TARGET} PRIVATE ggml)
+target_link_libraries(${TEST_TARGET} PRIVATE ggml pthread)
```

Comment:
This will not work on every OS, eg. it fails on windows. It's not clear why this is needed in your system - `test-backend-ops` does not use pthread, it uses the C++ standard library. If it is really necessary, this may need to be changed to use the `Threads` package instead.